ossfuzz/compress_frame_fuzzer.c

```diff
@@ -19,19 +19,20 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
 {
     FUZZ_dataProducer_t *producer = FUZZ_dataProducer_create(data, LZ4_compressBound(size));
     LZ4F_preferences_t const prefs = FUZZ_dataProducer_preferences(producer);
-    size_t const compressBound = LZ4F_compressFrameBound(FUZZ_dataProducer_remainingBytes(producer), &prefs);
-    size_t const dstCapacitySeed = FUZZ_dataProducer_uint32_seed(producer, 0, compressBound);
-    size_t const dstCapacity = FUZZ_dataProducer_uint32(dstCapacitySeed,
-        0, FUZZ_dataProducer_remainingBytes(producer));
+
+    size = FUZZ_dataProducer_remainingBytes(producer);
```

Comment:
`size` should be set only once, and after all extraction of `producer` are completed.
`compressBound` should depend on final `size`, not an intermediate version that will be updated later on.

I suspect the issue is you want to provide `compressBound` to retrieve `dstCapacitySeed` with `FUZZ_dataProducer_uint32()`. See my comment on this function. Without a need to provide `bounds` as parameters, it removes the need to determine `compressBound` before `dstCapacitySeed`.